app/src/lib/helpers/default-branch.ts

```diff
@@ -1,15 +1,29 @@
 import { getGlobalConfigValue, setGlobalConfigValue } from '../git'
 import { enableDefaultBranchSetting } from '../feature-flag'
 
+/**
+ * The default branch name that Desktop's embedded version of Git
+ * will use when initializing a new repository.
+ */
 export const DefaultBranchInGit = 'master'
 
+/**
+ * The default branch name that GitHub Desktop will use when
+ * initializing a new repository.
+ */
+export const DefaultBranchInDesktop = 'main'
+
+/**
+ * The name of the Git configuration variable which holds what
+ * branch name Git will use when initializing a new repository.
+ */
 const DefaultBranchSettingName = 'init.defaultBranch'
 
 /**
  * The branch names that Desktop shows by default as radio buttons on the
  * form that allows users to change default branch name.
  */
-export const SuggestedBranchNames: ReadonlyArray<string> = ['master', 'main']
+export const SuggestedBranchNames: ReadonlyArray<string> = ['main', 'master']
```

Comment:
```suggestion
export const SuggestedBranchNames: ReadonlyArray<string> = [DefaultBranchInDesktop, DefaultBranchInGit]
```
Not sure if this reduces "magic strings" or not